### DISCUSSION TITLE
MeshCentral not listening on port 482 for web and clients

### DISCUSSION THREAD

**Discussion by retechp** [Ylianst/MeshCentral#4167] [General]
Just hopped on the MeshCentral bandwagon :)

I have it installed and trying to get it to listen on non-standard ports.

It is running on Ubuntu 20.04
Node is v 10.19.0
NPM is v 6.14.4

The server starts but does not listen port 482

**_● meshcentral.service - Support
     Loaded: loaded (/etc/systemd/system/meshcentral.service; enabled; vendor preset: enabled)
     Active: active (running) since Thu 2022-06-23 14:46:50 UTC; 3h 38min ago
   Main PID: 668 (node)
      Tasks: 22 (limit: 1066)
     Memory: 197.9M
     CGroup: /system.slice/meshcentral.service
             ├─668 /usr/bin/node /home/mesh/node_modules/meshcentral --cert remote.mydomain.com --port 482
             └─875 /usr/bin/node /home/mesh/node_modules/meshcentral --cert remote.mydomain.com --port 482>

Jun 23 14:46:50 mesh01 systemd[1]: Started Remote Support.
Jun 23 14:46:59 mesh01 node[668]: MeshCentral HTTP redirection server running on port 80.
Jun 23 14:46:59 mesh01 node[668]: MeshCentral v1.0.39, Hybrid (LAN + WAN) mode, Production mode._**

- - - My Netstat -an output - - -
![ports](https://user-images.githubusercontent.com/108091170/175369435-02de4fad-137d-43af-bb9b-28ae28f14683.jpg)

- - - My systemctl service configuration - - -
![meshfile](https://user-images.githubusercontent.com/108091170/175370427-ed9a0537-796c-42c0-bcd6-4b21b53faecc.jpg)

**Comment by si458** [comment]
its best to config the ports from within the config.json rather than the command line
```
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "settings": {
    "cert": "myserver.com",
    "port": 482,
    "redirPort": 80,
    "RelayPort": 453
  }
}
```
